web/templates/premium.html

```diff
@@ -113,7 +113,7 @@ <h2 class="ui header">
                 <input
                   type="hidden"
                   name="custom"
-                  value='{"userid":"{{ $global.User.ID }}"}' />
+                  value="userid={{ $global.User.ID }}" />
```

Comment:
wtf why did i put this here

Reply:
highly based parsing code: https://github.com/osuAkatsuki/payments-service/blob/main/app/api/webhooks/paypal.py#L246

Reply:
what the fuck HAHA